src/github.com/getlantern/flashlight/README.md

```diff
@@ -45,7 +45,7 @@ Usage of flashlight:
   -portmap=0: try to map this port on the firewall to the port on which flashlight is listening, using UPnP or NAT-PMP. If mapping this port fails, flashlight will exit with status code 50
   -registerat="": base URL for peer DNS registry at which to register (e.g. https://peerscanner.getiantem.org)
   -role="": either 'client' or 'server' (required)
-  -server="": FQDN of flashlight server when running in server mode (required)
+  -frontfqdns="": YAML string representing a map from the name of each front provider to a FQDN that will reach this particular server via that provider (e.g. '{cloudflare: fl-001.getiantem.org, cloudfront: blablabla.cloudfront.net}')
```

Comment:
Just curious: Is passing a YAML string somehow preferable to passing a string with '=' as the character delimiter? i.e. 
```-frontfqdns cloudflare=fl-001.getiantem.org cloudfront=blablabla.cloudfront.net```?